assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -117,18 +116,21 @@ const List = styled(Area).attrs({
   overflow-x: scroll;
   overflow-y: hidden;
   margin: 0 0 ${CAROUSEL_BOTTOM_SCROLL_MARGIN}px 0;
-  scrollbar-color: rgba(255, 255, 255, 0.54) transparent;
-  scrollbar-width: auto;
-  &::-webkit-scrollbar {
-    width: ${SCROLLBAR_HEIGHT}px;
-    height: ${SCROLLBAR_HEIGHT}px;
-  }
+
+  /*
+   * These overrides are an exception - generally scrollbars should all
+   * look the same. We do this only here because this scrollbar is always visible.
+   */
+  scrollbar-color: ${({ theme }) => theme.colors.bg.v10}
+    ${({ theme }) => theme.colors.bg.v1} !important;
```

Comment:
Don't these work without `!important`? 😕

Reply:
sadly no - I'm no fan of important but couldn't make it work without. I don't know if styled components is to blame or if this is a bug with these pseudo elements..